working/v3.0.0-rc2/hybrid-flow/authorization-request-schema.json

```diff
@@ -131,6 +131,35 @@
       "default": 0,
       "examples": [86400]
     },
+    "display": {
+      "$id": "#/properties/display",
+      "type": "string",
+      "title": "Specify how to display authentication and consent to customer",
+      "enum": ["page", "popup", "touch", "wap"]
+    },
+    "prompt": {
+      "$id": "#/properties/prompt",
+      "type": "string",
+      "title": "Specify whether the authorisation server should prompt for authentication and consent to customer",
+      "examples": ["none", "login consent", "consent", "select_account"]
+    },
+    "ui_locales": {
+      "$id": "#/properties/ui_locales",
+      "type": "string",
+      "title": "Specify preferred languages to display authentication and consent to customer",
+      "examples": ["en-nz", "mi", "en-nz mi"]
+    },
+    "id_token_hint": {
+      "$id": "/properties/id_token_hint",
```

Comment:
Spotted this after clicking approve sorry, this I am pretty sure should be `#/properties/id_token_hint` inline with the other `$id` values.

Reply:
Good spotting, I've corrected that now and done a push to update